### PR TITLE
add stories for pagination keyline new props

### PR DESCRIPTION
**Pull request by rogeriopvl**
Add missing stories for `hideTopKeyline` and `hideBottomKeyline` props in pagination component.

### CODE REVIEW

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://www.bitrise.io/build/91e3a378e92e9c33

File: packages/pagination/pagination.stories.js [hunk 1]
```diff
@@ -60,6 +60,24 @@ storiesOf("Pagination", module)
       onPrev={action("last-page-prev-compact")}
     />
   ))
+  .add("Without top keyline", () => (
+    <Pagination
+      page={2}
+      count={60}
+      hideTopKeyline
+      onNext={action("last-page-next-compact")}
```
Comment (toboid) [inline packages/pagination/pagination.stories.js]: Small thing but it looks like the action names in this file reflect the story, this action name is for the story above

Comment (rogeriopvl) [reply packages/pagination/pagination.stories.js]: Oops, missed that! 👍

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://www.bitrise.io/build/e83881a7a4241c03

**Pull request merged**